src/main/kotlin/minesweeper/domain/game/Cell.kt

```diff
@@ -1,3 +1,7 @@
 package minesweeper.domain.game
 
-data class Cell(val type: CellType)
+data class Cell(val type: CellType, var count: Int = 0) {
+    companion object {
+        fun of(isMine: Boolean, count: Int): Cell = if (isMine) Cell(CellType.MINE) else Cell(CellType.NORMAL, count)
```

Comment:
```suggestion
        fun of(isMine: Boolean, count: Int): Cell = if (isMine) {
            Cell(CellType.MINE) 
        } else {
            Cell(CellType.NORMAL, count)
        }
```
코틀린에서는 if-else문은 반드시 중괄호를 사용합니다.
ref. https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements